lib/SPIRV/SPIRVReader.h

```diff
@@ -103,6 +103,10 @@ class SPIRVToLLVM {
   Instruction *transSGSizeQueryBI(SPIRVInstruction *BI, BasicBlock *BB);
   bool transFPContractMetadata();
   bool transKernelMetadata();
+  /*InlineAsm*/ Value *transAsmINTEL(SPIRVAsmINTEL *BA, Function *F,
```

Comment:
What is the meaning of `/*InlineAsm*/` comment? It looks a bit strange to me

Reply:
Leftover. Removed in latest commit

Reply:
It was telling that Value is actually is an InlineAsm...